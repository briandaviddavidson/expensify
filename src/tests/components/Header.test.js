import { shallow } from 'enzyme';
import React from 'react';
import Header from '../../components/Header';

test('see header tag in Header', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
})
