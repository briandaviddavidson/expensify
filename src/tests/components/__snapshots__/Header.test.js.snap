// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`see header tag in Header 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="isActive"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="isActive"
    ariaCurrent="true"
    to="/create"
  >
    Create
  </NavLink>
  <NavLink
    activeClassName="isActive"
    ariaCurrent="true"
    to="/help"
  >
    Help
  </NavLink>
</header>
`;
